app/views/public_pages/stimulus.html.erb

```diff
@@ -20,7 +20,7 @@
           broken down some common questions to help you out.
         </p>
         <p class="hint last">
-          The IRS will post all key information on <%=link_to 'https://irs.gov/eip', target: '_blank', "data-track-click": "stimulus-irs-eip", "data-track-attribute-position": "intro" do %>IRS.gov/eip<% end %>
+          The IRS will post all key information on <%=link_to 'https://irs.gov/eip', target: '_blank', rel: "noopener", "data-track-click": "stimulus-irs-eip", "data-track-attribute-position": "intro" do %>IRS.gov/eip<% end %>
```

Comment:
There is some risk that this might block click tracking. Lou, have you already checked?

Reply:
i will verify before we merge. 

noopener is different from noreferrer, it should only prevent the opened window from having access to the opening window. but could call to verify this.

Reply:
yup still works